### ISSUE TITLE
Weights File for the prediction.

### ISSUE THREAD

**Issue by haris-ali2** [dettmar/im2height#1]
Hello, I wanted to try to produce DSM predictions of satellite imagery using the pre-trained model. Do you know any way to get the weight file (.ckpt) for test predictions? Thank you

**Comment by dettmar** [comment]
Hi! No unfortunately I don't have any pre-trained weights as this was used for a company project. Nor did I reach out to the original authors. Maybe contact them to see if they have any weights available to share.

**Comment by haris-ali2** [comment]
@dettmar Thank you for the reply. Can you kindly tell me about the input data then? I am trying to train my own model. My inputs are RGB satellite images converted to ".npy". But when I try to train, it says 'operands could not be broadcasted together with remapped shapes [original ->remapped]: (3,2) and requested shape (2,2)'[error in line 64 in data.py]. I searched the internet and it says that the input should be RGB but I actually have RGB as inputs. My labels are grayscale 1 channel DSM images. It gave me an error before saying 'np.rollaxis should have a start value between -2 and 2 [error in line 53, 60 and 97 in data.py] but you had it hardcoded as 3. Can you tell me how did you manage to run that? I actually changed that value to 2 and it's okay now but can you kindly share with me the solution to the broadcasting problem if you have any insights? Thank you.

**Comment by dettmar** [comment]
This was a while ago, but we only ever used it on 1 channel in -> 1 channel out. My guess would be that pytorch is expecting a separate dim of channels and I might have missed this using only one channel. I honestly never had a serious look at it before uploading it here, so there might hard coded misses that were specifically used for our project. Unfortunately I don't have time looking through the project at the moment, but if you find out what's causing the issue, please feel free to send a PR and I'll review.

**Comment by haris-ali2** [comment]
Okay thanks. Will do. And yes, the issue was I was passing images of size (500,500) while it was expecting them to be of (500,500,1)
